specs/phase0/beacon-chain.md

```diff
@@ -788,10 +788,12 @@ def compute_activation_exit_epoch(epoch: Epoch) -> Epoch:
 #### `compute_domain`
 
 ```python
-def compute_domain(domain_type: DomainType, fork_version: Version=GENESIS_FORK_VERSION) -> Domain:
+def compute_domain(domain_type: DomainType, fork_version: Optional[Version]=None) -> Domain:
     """
     Return the domain for the ``domain_type`` and ``fork_version``.
     """
+    if fork_version is None:
```

Comment:
Looks good, however, with the new phase 1 changes the module is reloaded fully, instead of just reloading variables and type declarations. So this wouldn't be a problem in the future. Still think it is better practice to do this, and v0.10.1 needs it, so LGTM.